homework/tests/homework.spec.js

```diff
@@ -0,0 +1,230 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+const data = [
+  'Prototype',
+  '1',
+  '2',
+  '3',
+  '4',
+  '5',
+  '6',
+  '7',
+  '8',
+  '9'
+]
+
+
+data.forEach(version => {
+  test.describe(version + ': Add', () => {
```

Comment:
`test.describe` is used for tests block. (here we declare name for all tests that will be written in the following block)
`test()` is used for specific test.  And in tests results report we see tests by the joint name of `test.describe` and `test()`

In this case we see in report `Add Multiply...` which gives confusion which exactly operation are we really testing.
![image](https://user-images.githubusercontent.com/42140136/181120932-331bcb89-9a5a-4eea-b211-75156feef756.png)

Please adjust by having `test.describe` blocks for each operation.